src/main/java/io/cdap/plugin/successfactors/source/config/SuccessFactorsPluginConfig.java

```diff
@@ -45,9 +45,10 @@ public class SuccessFactorsPluginConfig extends PluginConfig {
   private static final String REFERENCE_NAME = "referenceName";
   private static final String REFERENCE_NAME_DESCRIPTION = "This will be used to uniquely identify this source/sink " +
     "for lineage, annotating metadata, etc.";
-  private static final String ASSOCIATED_ENTITY_NAME = "associatedEntityName";
+  public static final String ASSOCIATED_ENTITY_NAME = "associatedEntityName";
   private static final String NAME_SCHEMA = "schema";
   private static final String PAGINATION_TYPE = "paginationType";
+  public static final String EXPAND_OPTION = "expandOption";
```

Comment:
Ideally these variables should be only for internal use.

Reply:
But these are used in other classes as well.